Makefile

```diff
@@ -119,18 +119,11 @@ targets:	## Display available targets and descriptions
 
 ### Versioning Targets ###
 
-# Bumpversion 'dry' config
-# if 'dry' is specified as target, any bumpversion call using 'BUMP_XARGS' will not apply changes
-BUMP_XARGS ?= --verbose --allow-dirty	## Extra arguments to pass down to version control utility
-BUMP_XARGS := $(call clean_opt,$(BUMP_XARGS))
-ifeq ($(filter dry, $(MAKECMDGOALS)), dry)
-  BUMP_XARGS := $(BUMP_XARGS) --dry-run
-endif
```

Comment:
Revert the changes to deal with extra targets. This will cause `make bump dry` to not be correctly detected. Leave also the `BUMP_XARGS` logic. This is useful when working with intermediate releases or to check anticipated bump results without applying them. 

The `--allow-dirty` can be moved under the `ifeq` with `--dry-run`, since that is where it is most often used (check the result without having it committed).